src/controller/user-controller.js

```diff
@@ -0,0 +1,51 @@
+const client = require('../db');
```

Comment:
while i love what you did here, i want you to use a class then when exporting you can initialize. Also name this file camel case - userController.js